frontend/javascripts/oxalis/view/right-border-tabs/starting_job_modals.js

```diff
@@ -268,3 +310,78 @@ export function NeuronInferralModal({ handleClose }: Props) {
     />
   );
 }
+
+export function ApplyMergerModeModal({ handleClose }: Props) {
+  const dataset = useSelector(state => state.dataset);
+  const tracing = useSelector(store => store.tracing);
+  return (
+    <StartingJobModal
+      handleClose={handleClose}
+      jobName="apply merger mode"
+      suggestedDatasetSuffix="with_merged_segmentation"
+      chooseSegmentationLayer
+      jobApiCall={async (newDatasetName, segmentationLayer) => {
+        const volumeLayerName =
+          segmentationLayer.tracingId != null
+            ? getReadableNameByVolumeTracingId(tracing, segmentationLayer.tracingId)
+            : null;
+        const baseSegmentationName = getBaseSegmentationName(segmentationLayer);
+        return startApplyMergerModeJob(
+          dataset.owningOrganization,
+          dataset.name,
+          baseSegmentationName,
+          volumeLayerName,
+          newDatasetName,
+          tracing.annotationId,
+          tracing.annotationType,
+        );
+      }}
+      description={
+        <p>
+          Start a job that take the current state of this merger mode tracing and apply it to the
+          segmentation layer. This will create a new dataset which contains the merged segmentation
+          layer. If this dataset has more than one segmentation layer, please select the
+          segmentation layer to the merging should be applied to.
```

Comment:
```suggestion
          segmentation layer to which the merging should be applied to.
```

Reply:
on a second thought: when the merger mode is active, there can only be one visible segmentation layer. wouldn't simply using that layer make sense? one could still show the layer name in the modal (but as a disabled field) to inform the user. but making the user choose the layer again seems unnecessary to me?

Reply:
> when the merger mode is active, there can only be one visible segmentation layer.

In general, there con only be a single visible segmentation layer at a time. But it is possible to switch between segmentation layers while the merger mode is on.

Nevertheless, I applied you feedback. But I am still unsure whether to keep the value for the selected segmentation layer fixed. Because what if the user changes the visible segmentation layer before trying to trigger the merger mode job. Then the user would be unable to select the annotation layer they actually want the merging to be applied to. I know this is a pretty hypothetical scenario, but it is currently still possible and at least worth considering imo.

Reply:
> Because what if the user changes the visible segmentation layer before trying to trigger the merger mode job. Then the user would be unable to select the annotation layer they actually want the merging to be applied to. I know this is a pretty hypothetical scenario, but it is currently still possible and at least worth considering imo.

As long as it's transparent, on which layer the job will work, I think this is fine. The user can simply close the modal, change the visibility and open it again.

Reply:
Alrighty :D
In that case, all your requested changes should be applied